lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1247,18 +1262,24 @@ export class UserAgentApplication {
                 window.location.assign("/");
                 return;
             } else {
+                this.logger.verbose("Valid login request url obtained from cache");
                 const currentUrl = UrlUtils.removeHashFromUrl(window.location.href);
                 const finalRedirectUrl = UrlUtils.removeHashFromUrl(loginRequestUrl);
                 if (currentUrl !== finalRedirectUrl) {
+                    this.logger.verbose("Current url is not login request url, navigating");
```

Comment:
Can we add a PII log stating what both currentUrl and finalRedirectUrl are? This would help with debugging redirect issues.

Reply:
Sure thing! Added.